apps/client/src/features/viewers/studio/StudioClock.tsx

```diff
@@ -83,16 +85,28 @@ export default function StudioClock(props: StudioClockProps) {
       <NavigationMenu />
       <ViewParamsEditor paramFields={studioClockOptions} />
       <div className='clock-container'>
-        <div className={`studio-timer ${showSeconds ? 'studio-timer--with-seconds' : ''}`}>{clock}</div>
+        <div className='studio-timer-warp'>
+          <div className={`studio-timer ${showSeconds ? 'studio-timer--with-seconds' : ''}`}>{clock}</div>
+        </div>
         <div
           ref={titleRef}
           className='next-title'
           style={{ fontSize: titleFontSize, height: '10vh', width: '100%', maxWidth: '75%' }}
         >
-          {eventNext?.title ?? ''}
+          {eventNext?.title ?? ' '}
         </div>
-        <div className={isNegative ? 'next-countdown' : 'next-countdown next-countdown--overtime'}>
-          {selectedId !== null && formatDisplay(time.current)}
+        <div className='next-countdown-warp'>
+          <div
+            className={cx([
+              'next-countdown',
+              isNegative ? ' next-countdown--overtime' : '',
+              isPaused ? ' next-countdown--paused' : '',
+            ])}
+          >
+            {/* ! is used an a blank character //TODO: placeholder moves a bit when becomming active*/}
+            {isNegative ? '-' : '!'}
```

Comment:
I wonder if it is best not to have the `-` symbol. This would mean the test moves a bit, but that is maybe OK.
Most of the time the `-` will not be there, and the clock will be off-centre, so I wonder if we are sacrificing the common case for an edge case